commons/tools/warp10.md

```diff
@@ -146,3 +146,31 @@ An example where it is needed to increase the fetch limit by the `LIMIT` functio
 Quantum is a web tool used to run some warpscript. You can access to it from your metrics interface.
 
 It provide the path to the Clever Cloud warp10 gateway and let you explore your data.
+
+## Macro
+
+Clever Cloud provides some macros.
```

Comment:
Cloud you explain what macros are, with some links and documentation from warp10?